content/questions/actions/question-076.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "How many reusable workflows can you call from a single workflow file? "
+question: "Question 076"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations
+1. [] 0
+1. [] 1
+1. [x] 20
+1. [] 10
```

Comment:
Need's spaces in empty brackets and I've made it a standard to have the first answers to be correct. Doesn't change anything but keeps the repo consistent

```suggestion
1. [x] 20
1. [ ] 5
1. [ ] 1
1. [ ] 10
```